developer_manual/app/code_signing.rst

```diff
@@ -130,7 +130,7 @@ examples will assume that you are trying to sign an application named
    --certificate=/Users/lukasreschke/CA/contacts.crt --path=/Users/lukasreschke/Programming/contacts``
 
 The occ tool will store a ``signature.json`` file within the ``appinfo`` folder 
-of your application. Then compress the application folder and upload it to 
+of your application. Then compress the application folder (``contacts.tar.gz``) and upload it to 
```

Comment:
Only a **gziped tar file** is accepted here. But since there are [other algorithms](https://en.wikipedia.org/wiki/List_of_archive_formats#Archiving_and_compression) out there, it should be clear wich one is to be used.